src/platform/packages/shared/kbn-monaco/src/languages/esql/language.ts

```diff
@@ -32,7 +33,15 @@ export const ESQL_AUTOCOMPLETE_TRIGGER_CHARS = ['(', ' ', '[', '?'];
 
 export type MonacoMessage = monaco.editor.IMarkerData & { code: string };
 
-export const ESQLLang: CustomLangModuleType<ESQLCallbacks, MonacoMessage> = {
+export type ESQLDependencies = ESQLCallbacks &
```

Comment:
Use the same type I suggested in esql_editor

Reply:
I was thinking to move these types on the esql-types package and reuse them in the editor and monaco package